synedrion/src/uint/traits.rs

```diff
@@ -196,11 +215,11 @@ impl HasWide for U2048 {
     fn square_wide(&self) -> Self::Wide {
         self.square_wide().into()
     }
-    fn into_wide(self) -> Self::Wide {
-        (self, Self::ZERO).into()
+    fn to_wide(&self) -> Self::Wide {
+        Uint::concat_mixed(self, &Self::ZERO)
```

Comment:
Why do you prefer these lower level calls to the `From` impls? Just do be more explicit?

Reply:
If you're talking about the body of the function, it's because the `From` takes the arguments by value, while `concat_mixed()` takes a reference. So there's less chance of leaving secret stuff on the stack.

Reply:
aha, ok, got it. Ty!